draft/draft-ietf-bfd-optimizing-authentication.xml

```diff
@@ -392,12 +395,56 @@
         excepting that Auth Type is still TBD and that Reserved is set to 2.
       </t>
 
-      <section title="Error Handling">
+      <section title="Receipt Using Optimized Authentication">
+        <t>
+	  The procedures for authenticating BFD Control packets using Optimized
+	  Authentication is similar to the existing procedures covered in 
+	  <xref target="RFC5880" section="6.7"/>. 
+          Optimized Authentication modes have common procedural requirements for 
+	  authentication regardless of which strong and less computationally
+	  intensive authentication modes are used.
+        </t>
+        <t>
+	  The required value of the Auth Len field for a given Optimized
+	  Authentication mode is defined in the respective specifications for
+	  the strong mode and less computationally intensive mode.
+        </t>
+        <t>
+          The following common procedures apply to authenticating BFD Control
+          packets utilizing Optimized Authentication:
+        </t>
+        <t>
+          If the received BFD Control packet does not contain an 
+	  Authentication Section (<xref target="RFC5880" sectionFormat="comma" section="4.1"/>), or
+	  the Auth Type is not a supported Optimized Authentication Auth Type,
+	  then the received packet MUST be discarded.
+        </t>
         <t>
 	  If the received BFD Control packet contains an optimized
 	  authentication type using these procedures and the Optimized
           Authentication Mode field is not 1 or 2, then the received packet
           MUST be discarded.
+        </t>
+        <t>
+          If the Auth Len field is not equal to a value appropriate for the
+          Optimized Authentication Mode field, the packet MUST be discarded.
+        </t>
+        <t>
+          If bfd.AuthSeqKnown is 1, examine the Sequence Number field.  If the
+          sequence number lies outside of the range of bfd.RcvAuthSeq+1 to
+          bfd.RcvAuthSeq+(3*Detect Mult) inclusive (when treated as an unsigned
+          32-bit circular number space, the received packet MUST be discarded.
```

Comment:
```suggestion
          bfd.RcvAuthSeq+(3*Detect Mult) inclusive (when treated as an unsigned
          32-bit circular number space) the received packet MUST be discarded.
```
Missing a closing bracket?

Reply:
Pushed a fix.  Note that we have this typo in RFC 5880. :-)